<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScenesList</class>
 <widget class="QDialog" name="ScenesList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Cutscene Name:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="2">
    <widget class="QComboBox" name="language_comboBox"/>
   </item>
   <item row="8" column="2">
    <widget class="QListView" name="object_listView">
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="addScene_button">
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Add New Scene</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="name_lineEdit"/>
   </item>
   <item row="8" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="up_pushButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources/icons/icons.qrc">
         <normaloff>:/toolbar_icons/arrow-up.png</normaloff>:/toolbar_icons/arrow-up.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources/icons/icons.qrc">
         <normaloff>:/toolbar_icons/arrow-left.png</normaloff>:/toolbar_icons/arrow-left.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources/icons/icons.qrc">
         <normaloff>:/toolbar_icons/arrow-right.png</normaloff>:/toolbar_icons/arrow-right.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="down_pushButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources/icons/icons.qrc">
         <normaloff>:/toolbar_icons/arrow-down.png</normaloff>:/toolbar_icons/arrow-down.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <widget class="QTableView" name="scenes_tableView">
     <property name="editTriggers">
      <set>QAbstractItemView::AllEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QComboBox" name="sceneTypeSelector">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Clear Screen</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Clear Area</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Show/Move Image</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Move Viewpoint Image</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Show Image Animation</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Play Sound Effect</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Play Music</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Stop Music</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Text</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sub-Scene</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Parallax</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Parts:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Language:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="sceneSelector"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/icons/icons.qrc"/>
  <include location="../resources/icons/icons.qrc"/>
  <include location="../resources/icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
